weaver/core/network/corda-interop-app/interop-workflows/src/main/kotlin/org/hyperledger/cacti/weaver/imodule/corda/flows/ViewVerification.kt

```diff
@@ -117,16 +117,20 @@ fun verifyCordaNotarization(viewData: ByteString, verificationPolicyCriteria: Li
     println("Corda view data: $cordaViewData")
     
     var interopPayload = cordaViewData.notarizedPayloadsList[0].payload
+    var signersMut = mutableListOf<String>()
 
     // 2. Map over the list of notarizations and verify the signature, creating a list of Either Error Boolean
     val eitherErrorCordaViewData = cordaViewData.notarizedPayloadsList.map { notarizedPayload ->
         getCertificateFromString(notarizedPayload.certificate).flatMap { x509Cert ->
             // 3. Check the certificates are valid according to the [Membership].
+            println("Verify membership of signer")
             verifyMemberInSecurityDomain(x509Cert, securityDomain, notarizedPayload.id, serviceHub).flatMap {
+                println("Verify signature")
```

Comment:
Maybe add some specific identity info to these print statements, if we are using them for logging and debugging.